css/main-title.css

```diff
@@ -0,0 +1,45 @@
+.title{
```

Comment:
cái này không phải của riêng title nha e. e nhìn toàn bộ design rất nhiều chỗ có khoảng trắng 2 đầu.
a dặn tách ra 2 class rồi mà :v